projects/canopy/src/lib/docs/principles/accessibility/overview.stories.mdx

```diff
@@ -0,0 +1,210 @@
+import { Meta } from '@storybook/addon-docs';
+import Feedback from '../../common/feedback.md';
+
+<Meta title="Principles/Accessibility/Overview" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Overview
+
+> "The power of the Web is in its universality.
+> Access by everyone regardless of disability is an essential aspect."
+>
+> <cite>Tim Berners-Lee, W3C Director and inventor of the World Wide Web</cite>
+
+## Introduction
+
+Before diving into explaining some of the key aspects of web accessibility here are some interesting facts to make you understand how important it is to create websites that are accessible by any user no matter their disabilities:
+
+- there are around 14.6 million disable people in the UK (almost one in five of the population).
+- only 17% of disabled people were born with their disabilities. The majority of disabled people acquire their disability later in life.
+- for an average business, disabled people may account for up to 20% of all costumers. Many businesses fail to recognise the business opportunity that improved access presents.
+- the main reasons for disabled customers switching to a more accessible competitor include inaccessible websites.
+
+## What is Web accessibility?
+
+Web accessibility means that websites, tools, and technologies are designed and developed so that people with disabilities can use them.
+
+Web accessibility isn’t just a moral requirement but also a legal one:
+
+> [...] website owners have been obliged by law to ensure that their websites are accessible to all users, as it is illegal to treat those with disabilities less favourably.
+>
+> <cite>Equality Act of 2010</cite>
+
+In order to achieve this we have to start thinking and learning about how other users use our websites and applications; in what circumstances do they use it.
```

Comment:
```suggestion
In order to achieve this we have to start thinking and learning about how all users use our websites and applications, and in what circumstances they use them.
```

I don't really like 'othering' people who use assistive technologies. I know that wasn't the intent, of course :)